tvpeter/automart
# #167012866 : User creates ad UI and API integration

#### What does this PR do?

- Connects the User create ad front end with the backend API

#### Description of Task to be completed?

- Validate that a user is signed in to access the page

- Collect form data and submit to the api using fetch api

#### How should this be manually tested?

- Create an account with the signup

- Visit the create ad page to send data to the api

- it should create a car advert

#### Any background context you want to provide?

- Both the api and the UI were built separately

#### What are the relevant pivotal tracker stories?

[#167012866](https://www.pivotaltracker.com/story/show/167012866)
